Dockerfile

```diff
@@ -1,9 +1,28 @@
-FROM ruby:2.2.3-slim
-
-RUN apt-get update && apt-get install -y wget apt-transport-https git
-RUN wget -qO- https://deb.nodesource.com/gpgkey/nodesource.gpg.key | apt-key add -
-RUN echo 'deb https://deb.nodesource.com/node_0.12 jessie main' > /etc/apt/sources.list.d/nodesource.list
-RUN apt-get update && apt-get install -y nodejs build-essential
+FROM ruby:2.2.4-alpine
+
+# install packages and clear the cache after
+# - nodejs for precompile and runtime asset compilation
+# - ca-certificates so we can do ssl
+# - git because we run checkouts
+# - nokogiri requirements
+# - binaries for all the databases we support
+RUN apk add --update \
```

Comment:
You can say `apk --no-cache add ...` 

> As of Alpine Linux 3.3 there exists a new --no-cache option for apk. It allows users to install packages with an index that is updated and used on-the-fly and not cached locally

https://github.com/gliderlabs/docker-alpine/blob/master/docs/usage.md#disabling-cache